diff -uNr imagination-2.1-orig/src/support.c imagination-2.1/src/support.c
--- imagination-2.1-orig/src/support.c	2009-10-07 02:10:04.000000000 -0400
+++ imagination-2.1/src/support.c	2009-10-24 22:01:59.000000000 -0400
@@ -20,7 +20,7 @@
 #include "support.h"
 #include <glib/gstdio.h>
 
-#define PLUGINS_INSTALLED 0
+#define PLUGINS_INSTALLED 1
 
 static gboolean img_plugin_is_loaded(img_window_struct *, GModule *);

